adversary.py

```diff
@@ -102,7 +102,8 @@ def try_sybil(self, guard):
 
         return False
 
-    def get_pwnage_time(self, guard):
+    def 
```

Comment:
Hey. This function definition is broken with this commit. Can you test it and resubmit? Thanks!